tests/test_pattern_rewriter.py

```diff
@@ -1,10 +1,9 @@
 from conftest import assert_print_op
 
-from xdsl.dialects import test
 from xdsl.dialects.arith import Addi, Arith, Constant, Muli
 from xdsl.dialects.builtin import Builtin, IntegerAttr, ModuleOp, StringAttr, i32, i64
-from xdsl.dialects.scf import If, Scf
-from xdsl.ir import Block, MLContext, Operation, Region
+from xdsl.dialects.test import Test, TestOp, TestType
```

Comment:
Once upon a time, we decided to try to make these look like this:
```
from xdsl.dialects import test
```

In this way, we can then write `test.testop` which looks a bit more like the SSA-form.
Not saying you should do this, just saying that you could, if you wanted to :wink:

Reply:
Hmm, I have no problem with this, it does look nicer to me as well.
Maybe that's something for documentation on code conventions? This is the first time I "hear" this.

Reply:
Yes, we should put this on our todo list. And opening a todo list should be on there too :thinking: